traitsui/testing/tester/wx/tests/test_helpers.py

```diff
@@ -33,13 +33,12 @@
 class TestInteractions(unittest.TestCase):
 
     def setUp(self):
-        self.app = wx.App()
         self.frame = wx.Frame(None)
         self.frame.Show()
 
     def tearDown(self):
-        wx.CallAfter(self.app.ExitMainLoop)
-        self.app.MainLoop()
```

Comment:
and without these lines, we will have a Wx Application object left behind at the end of the test run. Is that correct?

Reply:
Yep.

Reply:
I don't think any of these tests require an application though actually.

Reply:
We ask the event to be processed directly on the widget, I think.